scripts/corrections/make_theory_corr_ew.py

```diff
@@ -34,6 +34,9 @@
     hden = hh.normalize(res[f'{proc}_{args.den}']['output']['nominal_ew'])
     hratio = hh.divideHists(hnum, hden)
     hratio = hh.smoothTowardsOne(hratio)
+    scale = np.sum(hden.values()) / np.sum(hden.values()*hratio.values())
```

Comment:
Can use h.sum()

Reply:
If the problem is that it gives back a WeightedSum object, I think it's just hist.sum().value (or similar)